### ISSUE TITLE
[Suggestion] Overgrown Style: Add stylevar option to add chances of getting "Small vegetation" on Light Strips

### ISSUE THREAD

**Issue by LautaroL20** [TeamSpen210/BEE2-items#855]
![20160921182737_1](https://cloud.githubusercontent.com/assets/12244376/18734107/6acd0d24-8047-11e6-9ceb-3cafa2424cce.jpg)

**Comment by ZombiePwner1** [comment]
What map is that?

**Comment by LautaroL20** [comment]
I don´t remember now, but just wanted to give a simple example of the suggestion.

**Comment by Luke18033** [comment]
Why a stylevar? It doesn't really affect the lighting, seems like this could just be a random variant for ceiling light strips.

**Comment by LautaroL20** [comment]
> Seems like this could just be a random variant for ceiling light strips.

Indeed. I think that when i made this suggestion in 2016, i was a rookie and i can´t remember why i said it should be a "stylevar".
